examples/AccessControl/AccessControl.ino

```diff
@@ -23,9 +23,9 @@
   |         |              |           |                |                    |              |
   |    +----v-----+   +----v----+   +--v--------+     +-v----------+  +------v----+         |
   |    |MASTER TAG|   |KNOWN TAG|   |UNKNOWN TAG|     |GRANT ACCESS|  |DENY ACCESS|         |
-  |    +----------+   +---+-----+   +-----+-----+     +-----+------+  +-----+-----+         |
-  |                       |               |                 |               |               |
-  |       +----+     +----v------+     +--v---+             |               +--------------->
+  |    +-----+----+   +---+-----+   +-----+-----+     +-----+------+  +-----+-----+         |
+  |          |            |               |                 |               |               |
+  |       +--v-+     +----v------+     +--v---+             |               +--------------->
   +-------+EXIT|     |DELETE FROM|     |ADD TO|             |                               |
```

Comment:
Seems that there are two dead ends:

* ADD TO EEPROM   
* DELETE FROM EEPROM

please fix them too